VirtoCommerce.InventoryModule.Web/ExportImport/InventoryExportImport.cs

```diff
@@ -117,24 +132,13 @@ public void DoImport(Stream backupStream, Action<ExportImportProgressInfo> progr
                             var fulfillmentCentersType = AbstractTypeFactory<FulfillmentCenter>.TryCreateInstance().GetType().MakeArrayType();
                             var fulfillmentCenters = _jsonSerializer.Deserialize(reader, fulfillmentCentersType) as FulfillmentCenter[];
 
-                            progressInfo.Description = $"The {fulfillmentCenters.Count()} fulfillmentCenters have been imported";
+                            progressInfo.Description = $"The {fulfillmentCenters.Count()} fulfillmentCenters has been imported";
```

Comment:
`have`

Reply:
Done, in fact it was just revert to previous version.